code/game/objects/items/weapons/lighter.dm

```diff
@@ -17,6 +17,7 @@
 
 /obj/item/flame/lighter/Initialize()
 	. = ..()
+	item_flags |= ITEM_FLAG_HOLLOW
```

Comment:
Because I'm on the fence between doing this above, or just copying over the previous flag values and slapping them in the class definition. The way above means that any previously set bits in the parent would carry over unmolested.

Reply:
Also, I forgot to remove those in Initialize after I opted for the class definition. ^^;